WcaOnRails/db/migrate/20160811013347_add_announced_at_to_competitions.rb

```diff
@@ -0,0 +1,16 @@
+class AddAnnouncedAtToCompetitions < ActiveRecord::Migration
+  def up
+    add_column :Competitions, :announced_at, :datetime
+    execute <<-SQL
+      UPDATE Competitions
+      SET announced_at = (SELECT created_at
+        FROM posts
+        WHERE posts.title LIKE CONCAT(Competitions.name,'%')
+        AND posts.created_at IS NOT NULL and posts.created_at NOT LIKE '2007-07-03%')
```

Comment:
I saw a lot of competitions announced on that date and thought it might have been a bogus one, but looks like it's fine. I'll remove the condition.